docs/checks/invalidTagsCheck.md

```diff
@@ -1,17 +1,28 @@
 # Invalid Tags Check
 
-This check flags features based on configurable filters.   
+This check flags features for invalid tags based on a list of TaggableFilters. The list of TaggableFilters for this check is either a combination of filters passed through the 
+check configuration and default list of TaggableFilters or just the default filters. 
+The two configurable boolean values, "override.default.filters" and "append.to.default.filters", 
+determine the list of taggable filters that each of the atlas entities will be tested against. The rules are
+1) If "override.default.filters" is true and "append.to.default.filters" is false,
+only the filters that are passed through the configuration file will be used to test the atlas entities. 
+2) If "append.to.default.filters" is true and "override.default.filters" is false,
```

Comment:
If these filters are mutually exclusive then why not just have one switch?

Reply:
The check originally had empty default filters and I wanted to keep it that way and so used two switches. Since list of new filters added is long and couldn't be appended to the config file, I saved them in a resource file and made use of these two switches to either use the filters in the resource or add the config filters to the ones in resource.

Reply:
I think if we are adding a resource then it makes sense to have that as the default, but I like having the option to overwrite or append to it.

Reply:
Yeah the only reason these filters are not in the config is because there are many. These are solely water-island related tag combinations and need not be default filters.